hack/lib/test/junit.sh

```diff
@@ -158,7 +158,7 @@ function os::test::junit::generate_report() {
     if [[ -z "${JUNIT_REPORT_OUTPUT:-}" ||
           -n "${JUNIT_REPORT_OUTPUT:-}" && ! -s "${JUNIT_REPORT_OUTPUT:-}" ]]; then
         # we can't generate a report
-        return
+        return 0
```

Comment:
This is a no-op and should not be necessary

Reply:
Not returning 0 here was causing test-cmd to fail for me locally